soepy/pre_processing/model_processing.py

```diff
@@ -145,10 +145,12 @@ def read_model_spec_init(model_spec_init, model_params):
 
 
 def expand_model_spec_dict(model_spec_init_dict, model_params_df):
-    # Calculate range of years of education in the (simulated) sample
-    educ_min = model_spec_init_dict["INITIAL_CONDITIONS"]["educ_min"]
-    educ_max = model_spec_init_dict["INITIAL_CONDITIONS"]["educ_max"]
-    educ_range = educ_max - educ_min + 1
+    # Gather education years in list object
```

Comment:
Why not specify the `corresponding_educ_years` directly? This is my main question, as this requires switching between years and labels in the code ...

Reply:
Done. Adjusted the interface of the init file according to what we discussed yesterday.